azimuth_schedule_operator/models/v1alpha1/schedule.py

```diff
@@ -10,18 +10,18 @@ class ScheduleStatus(schema.BaseModel):
     ref_found: bool = False
     # updated when delete has been triggered
     delete_triggered: bool = False
```

Comment:
Not sure if I might prefer a single `phase` that transitions from `Pending` to `Waiting` (when we do the scheduled create case) to `ResourceAvailable` to `ResourceDeleted`?

Reply:
I couldn't really decide on what to do for those phases, as I wanted to be clear when the operator had noted it can access the ref, etc.

Reply:
Move to: ref_delete_triggered